RecoTracker/IterativeTracking/python/DetachedQuadStep_cff.py

```diff
@@ -201,6 +201,7 @@
 detachedQuadStepTrackCandidatesMkFitConfig = mkFitIterationConfigESProducer_cfi.mkFitIterationConfigESProducer.clone(
     config = 'RecoTracker/MkFit/data/mkfit-phase1-detachedQuadStep.json',
     ComponentName = 'detachedQuadStepTrackCandidatesMkFitConfig',
+    minPt = 0.075,
```

Comment:
I thought that this was supposed to be a technical PR wrt default pp reco.

Reply:
I could set it to zero. I set the same values as for CKF for the time being, since the effect is barely visible. But I can set it to zero.